tools/nl/embeddings/data/preindex/medium/duplicate_names.csv

```diff
@@ -16,7 +16,6 @@ Area_Farm_Producer_NativeHawaiianOrOtherPacificIslanderAlone,Count_Farm_Producer
 Count_Household_SingleMotherFamilyHousehold_With1Worker_BelowPovertyLevelInThePast12Months,Count_Household_SingleMotherFamilyHousehold_OwnerOccupied_BelowPovertyLevelInThePast12Months,Single Mother Family Households Below Poverty Level in The Past 12 Months
 Count_HousingUnit_HouseholderEducationalAttainmentBachelorsDegreeOrHigher_RenterOccupied,Count_HousingUnit_HouseholderEducationalAttainmentBachelorsDegreeOrHigher_OccupiedHousingUnit,Population With Bachelor's Degree or Higher Occupying Housing Units
 Count_HousingUnit_HouseholderRaceTwoOrMoreRaces_OwnerOccupied,Count_HousingUnit_HouseholderRaceTwoOrMoreRaces_RenterOccupied,Housing Units Occupied by Multiracial Population
-Count_MedicalConditionIncident_ConditionLymeDisease,Count_MedicalConditionIncident_ConditionLymeDisease_ConfirmedCase,number of lyme disease cases

```

Comment:
nit: Why are this and `sv_descriptions.csv` changing?

Can we do that separately?

Reply:
Removed. I believe these are coming from the changes made due to running the build_embeddings script